src/apps/testapps/testPolyfill.c

```diff
@@ -64,10 +64,10 @@ SUITE(polyfill) {
 
     TEST(maxPolyfillSize) {
         int numHexagons = H3_EXPORT(maxPolyfillSize)(&sfGeoPolygon, 9);
-        t_assert(numHexagons == 3169, "got expected max polyfill size");
+        t_assert(numHexagons == 7057, "got expected max polyfill size");
```

Comment:
This is quite a substantial change. Were we that far off previously?

I'm not sure if we can do this easily in the C library, but this is the kind of thing that would really benefit from visual snapshot testing. I'm going to take a look at rendering the two results here to verify - otherwise I'm really not sure how we can have any confidence in either number :/.

Reply:
Oh... Sorry, nevermind. I read this as the `polyfill` output, not the `maxPolyfillSize` output, which of course is expected to change substantially in this diff.

The comment about visual snapshot testing still applies, but not to this specific change.

Reply:
I'd appreciate adding a test case which fails for the `1.5` constant but works with `1.0`.

Reply:
@isaacbrodsky test added:

`src/apps/testapps/testPolyfill_GH136.c`